lab-cameron/.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
**TODO FOR LOST POINTS**

Just a top level comment that your ``index.js`` module should also have your functionality required in. I realize it's not clear, but the instructions of :

```
make a start script for running your index.js in node
```
...implies that the ``index.js`` module should have something in it.